spec/unit/lib/temporal/activity/context_spec.rb

```diff
@@ -3,31 +3,31 @@
 require 'temporal/scheduled_thread_pool'
 
 describe Temporal::Activity::Context do
-  let(:client) { instance_double('Temporal::Client::GRPCClient') }
+  let(:connection) { instance_double('Temporal::Connection::GRPC') }
```

Comment:
This turned out to be the wrong/a non-existent type. It's not strictly necessary for this PR, but I fixed it while I was in here.